bluetooth-permissions/src/jvmMain/kotlin/BluetoothPermissionManager.kt

```diff
@@ -1,5 +1,5 @@
 /*
- Copyright (c) 2020. Splendo Consulting B.V. The Netherlands
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
```

Comment:
🔍

Reply:
I've just checked in `.idea/copyright/Splendo.xml` and there is the text without `(c)`. The file is under version control.

Reply:
Indeed